rclcpp_action/test/test_client.cpp

```diff
@@ -480,23 +480,25 @@ TEST_F(TestClient, async_cancel_one_goal_with_callback)
   auto goal_handle = future_goal_handle.get();
   EXPECT_EQ(rclcpp_action::GoalStatus::STATUS_ACCEPTED, goal_handle->get_status());
 
-  bool cancel_response_received = false;
+  bool cancel_callback_received = false;
```

Comment:
@jacobperron nit: why changing the name?

Reply:
Twas an accident. Reverted 800a2cb